packages/backend/src/db/house/houseStore.service.spec.ts

```diff
@@ -87,6 +88,7 @@ describe('HouseStoreService', () => {
       address: 'Mars',
       code: 'lol',
       owner: null,
+      users: null,
```

Comment:
why null?

Reply:
````users```` is a list of objectID and is not specified/created. So should be set to null to make things equal.

Reply:
shouldn't it just be an empty list?

Reply:
Sorry didn't notice this reply yesterday. Empty list would work as well.